lib/uplinkc/uplink_definitions.h

```diff
@@ -32,6 +32,7 @@ typedef struct Downloader       { long _handle; } DownloaderRef;
 typedef struct Uploader         { long _handle; } UploaderRef;
 typedef struct EncryptionAccess { long _handle; } EncryptionAccessRef;
 typedef struct Scope            { long _handle; } ScopeRef;
+typedef struct Caveat           { long _handle; } CaveatRef;
```

Comment:
`Caveat` should be more like `EncryptionRestriction`. Struct with fields.

Reply:
You can for now only expose `Disallow*` fields in this struct.

Reply:
Done that!